### PR TITLE
Update my-credits display conditions

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Update `my-credits` exhibition conditions through Dreamstore, the Portal and the old version of the Portal.

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,34 @@ Watch mode:
 ```shell
 yarn test:watch
 ```
+
+## Development
+
+Note about `pages.json` and `my-credits`:
+
+- The first bloc is used by Dreamstore.
```
Comment (lbebber) [inline README.md]: Unless its usage is already established, the word "bloc" is more appropriate for like, groups of countries or political parties and such. I think "block" would be better in this case

File: react/ExtensionLinks.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types'
+import { intlShape, injectIntl } from 'react-intl'
+import { withRuntimeContext } from 'render'
+
+// Similar to ./index.js
```
Comment (lbebber) [inline react/ExtensionLinks.js]: nao sei se entendi essa questao de "similar to xxx", aqui e no outro arquivo. 

quais as similaridades? essas similaridades vao necessariamente permanecer? precisa desse comentario?

Comment (kevinch) [reply react/ExtensionLinks.js]: Deletei, já que coloquei informações no readme.

File: README.md (hunk 1 above)
Comment (kevinch) [reply README.md]: Simple french to english wrong translation. Good catch.

**Pull request merged**